.editorconfig

```diff
@@ -1,7 +1,7 @@
 [*.kt]
 indent_size=2
 insert_final_newline=true
-disabled_rules=import-ordering
+disabled_rules=import-ordering,experimental:package-name,experimental:argument-list-wrapping
```

Comment:
I disabled these two rules, which I thought were particularly overzealous. I'm open to disabling others if folks prefer.

https://github.com/pinterest/ktlint#experimental-rules